soc/renesas/rz/rzg3s/pinctrl_soc.h

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2024 Renesas Electronics Corporation
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_SOC_RENESAS_RZ_RZG3S_PINCTRL_SOC_H_
+#define ZEPHYR_SOC_RENESAS_RZ_RZG3S_PINCTRL_SOC_H_
+
+#include "../common/pinctrl_rzg.h"
```

Comment:
Should improve for include?

Reply:
Thank you.
I improve this include.
Please kindly check it